src/app_service/service/message/service.nim

```diff
@@ -581,6 +588,21 @@ QtObject:
     except Exception as e:
       error "error: ", procName="pinUnpinMessage", errName = e.name, errDesription = e.msg
 
+  proc asyncMarkMessageAsUnread*(self: Service, chatId: string, messageId: string) =
+    if (chatId.len == 0):
+      error "empty chat id", procName="markAllMessagesRead"
+      return
+
+    let arg = AsyncMarkMessageAsUnreadTaskArg(
+      tptr: cast[ByteAddress](asyncMarkMessageAsUnreadTask),
+      vptr: cast[ByteAddress](self.vptr),
+      slot: "onAsyncMarkMessageAsUnread",
+      requestId: $genUUID(),
```

Comment:
I don't see that you actually use this `requestId` anywhere, is it needed?

Reply:
Yeah, I thought we had metrics to put the request ID in, for now I'll have to remove it.